### ISSUE TITLE
0.7.x hangs when run with eth-gas-reporter

### ISSUE THREAD

**Issue by nmlinaric** [sc-forks/solidity-coverage#500]
Hello!

I'm trying to use `solidity-coverage` with `eth-gas-reporter` in a `buidler` project but I'm facing the same problem as described in this [issue](https://github.com/sc-forks/solidity-coverage/issues/423).

In the PR related with issue I see only files regarding `truffle` changed, maybe the issue exists when used with `buidler`?

Here's my [buidler.config.ts](https://pastebin.com/aQuXMdek) and [.solcover.js](https://pastebin.com/vVJEDyXC). Also tried it both with a `.ts` and `.js` - `buidler.config`.

**Comment by cgewecke** [comment]
@nmlinaric Thanks for reporting. 

There's should be a work-around by using the gas-reporter's [enabled][1] option.
ex:
```javascript
gasReporter: {
  coinmarketcap: process.env.COINMARKETCAP_API_KEY,
  enabled: (process.env.REPORT_GAS) ? true : false 
},
```

The command would be:
```
REPORT_GAS=false npx buidler coverage
```

[1]: https://github.com/cgewecke/buidler-gas-reporter#configuration

**Comment by nmlinaric** [comment]
@cgewecke Thanks for quick response.

With that workaround I managed to get working both `gas-reporter` and `solidity-coverage` but with a weird config:

 ```
- It doesn't matter if the `REPORT_GAS` variable in `.env.` is empty, true or false

 - REPORT_GAS=boolean npx buidler test  // doesn't matter if boolean is true or false but it has to be provided

 - npx buidler coverage  // must not have REPORT_GAS=boolean provided, if it is provided with any value (true/false) it hangs
```

**Comment by cgewecke** [comment]
@nmlinaric Hope you don't mind - re-opening because still need to fix this 🙂. The coverage tool should detect this case when using buidler and handle it.

**Comment by nmlinaric** [comment]
@cgewecke Sorry, thought maybe to close it because it's kinda working, but I agree it would be nice to have it working like it is supposed to be. 🙂

**Comment by cgewecke** [comment]
Should be fixed in `0.7.5` via #504
